packages/react/src/transactions/adapters/PublicationCallGateway.ts

```diff
@@ -168,7 +179,10 @@ export class PublicationCallGateway implements ICreatePostCallGateway {
       chainType: ChainType.POLYGON,
       id: v4(),
       request,
-      asyncRelayReceipt: this.initiatePostCreation(request),
+      asyncRelayReceipt: this.initiatePostCreation(
```

Comment:
nit: could we just leave the code as it was in `resolveCreatePostRequestArg` and just call that function over here? 
```
this.initiatePostCreation(await this.resolveCreatePostRequestArg(request))
```

Reply:
ah! that looks neater, I like it! Will have a look at such solution.

Reply:
Much neater! Good suggestion. See here: https://github.com/lens-protocol/lens-sdk/pull/61/commits/7c92d07195fc6f8be041351d0bf7c0785cc93d3b